monai/networks/nets/highresnet.py

```diff
@@ -0,0 +1,212 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch.nn as nn
+import torch.nn.functional as F
+
+from monai.networks.layers.convutils import same_padding
+from monai.networks.layers.factories import (get_conv_type, get_dropout_type, get_normalize_type)
+
+SUPPORTED_NORM = {
+    'batch': lambda spatial_dims: get_normalize_type(spatial_dims, is_instance=False),
+    'instance': lambda spatial_dims: get_normalize_type(spatial_dims, is_instance=True),
+}
```

Comment:
for https://github.com/Project-MONAI/MONAI/issues/81 would be great to have a factory to support group norm as well. @ericspod